app/controllers/volunteer_ops_controller.rb

```diff
@@ -69,36 +58,49 @@ def destroy
   end
 
   def volunteer_op_params
-    params.require(:volunteer_op).permit(
-      :description,
-      :title,
-      :organisation_id,
-      :address,
-      :postcode
-    )
+    args = [:description, :title, :organisation_id, :address, :postcode]
+    params.require(:volunteer_op).permit(*args)
   end
 
   private
 
+  def restrict_by_feature_scope
+    return :all if Feature.active?(:doit_volunteer_opportunities)
+    :local_only
+  end
+
   def authorize
     # set @organisation
     # then can make condition:
     # unless current_user.can_edit? organisation
-    unless org_owner? or superadmin?
-      flash[:error] = 'You must be signed in as an organisation owner or site superadmin to perform this action!'
-      redirect_to '/' and return
+    unless org_owner? || superadmin?
+      flash[:error] = 'You must be signed in as an organisation owner or ' \
+                      'site superadmin to perform this action!'
+      (redirect_to '/') && return
     end
   end
 
   def org_owner?
-    if params[:organisation_id].present? && current_user.present? && current_user.organisation.present?
+    if params[:organisation_id].present? && user_and_organisation_present?
       current_user.organisation.friendly_id == params[:organisation_id]
-    else
-      current_user.organisation == VolunteerOp.find(params[:id]).organisation if current_user.present? && current_user.organisation.present?
+    elsif user_and_organisation_present?
+      current_user.organisation == VolunteerOp.find(params[:id]).organisation
     end
   end
-  
+
+  def user_and_organisation_present?
```

Comment:
I wonder if this might be renamed `current_user_has_organisation?` or if we should add a `has_organisation?` method to user ... also I'm leaning towards redefining devise's `current_user` so it can't return nil, only a NilUser object, but that would be a separate ticket ...